### PR TITLE
README.md: Adding lookalike diffs

### PR DESCRIPTION
**Pull request by bphd**
## Summary by CodeRabbit

- **Documentation**
	- Added new sections in the README to reference differences between the current plugin and two other plugins.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,3 +213,9 @@ context.
 ##  Acknowledgements
 
 Aider CLI tool created by [Paul Gauthier](https://github.com/paul-gauthier).
+
+# `diff GeorgesAlkhouri/nvim-aider`
+// TODO: State differences
+
+# `diff joshuavial/aider.nvim`
+// TODO: State differences
```
[the remark this answers is not in the document]
Comment (bphd) [reply README.md:221]: @coderabbitai The comment part has been resolved on another suggestion. For the title part, vim community is very straight to the ground when it's about formatting, not very pretty, very "Canva", so I thought that quoting a Linux command would be more on the mood for such project. But at the end of the day that would be for @nekowasabi to decide, so I let that suggestion to him

File: README.md [hunk 2]
```diff
@@ -213,3 +213,9 @@ context.
 ##  Acknowledgements
 
 Aider CLI tool created by [Paul Gauthier](https://github.com/paul-gauthier).
+
+# `diff GeorgesAlkhouri/nvim-aider`
+// TODO: State differences
```
[the remark this answers is not in the document]
Comment (bphd) [reply README.md]: @codeRabbitAi see https://github.com/nekowasabi/aider.vim/pull/36?new_mergebox=true#discussion_r1896868986

[the remark this answers is not in the document]
Comment (bphd) [reply README.md]: @codeRabbitAi It rather states:
> For the title part, vim community is very straight to the ground when it's about formatting, not very pretty, very "Canva", so I thought that quoting a Linux command would be more on the mood for such project. But at the end of the day that would be for @nekowasabi to decide, so I let that suggestion to him

**Comment by nekowasabi** [comment]
Thanks for the PR.
It might be useful to display a comparison with similar plugins.
However, in the future, when there are more plugins, you will have to add and edit them, and you may also need to investigate whether there are any new plugins.
I'm sorry, but I don't have the time to do that.

If you think a comparison of plugins is necessary for you, I think it would be more useful if you published it as an article on your own website.

**Comment by bphd** [comment]
> article

Article goes outdated and aren't linked to what they study. And I still don't know differences between those plugins that claims to do the same thing. Installing one of them takes more than two weeks, so I guess I will never figure out

**Pull request closed without merging**